frontend/src/component/Activity/FolderActivity.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ContentActivityHeader from './ContentActivityHeader.jsx'
+
+export const FolderActivity = props => {
```

Comment:
Following the same idea of the other comment:
```suggestion
export const ContentWithoutPreviewActivity = props => {
```